.circleci/config.yml

```diff
@@ -29,7 +29,7 @@ commands:
       - run:
           name: Install pnpm package manager
           command: |
-            corepack enable
+            corepack enable --install-directory ~/bin
```

Comment:
Will this work if that directory doesn't exist?

Reply:
Yeah, it'll create it if it doesn't exist. We just don't have access to `/<os>` folders, but we should have access to the home directory.